Library/Homebrew/language/python.rb

```diff
@@ -92,18 +92,16 @@ def self.setup_install_args(prefix, python = "python3")
 
     # Mixin module for {Formula} adding shebang rewrite features.
     module Shebang
-      module_function
-
       # @private
-      def python_shebang_rewrite_info(python_path)
+      def self.python_shebang_rewrite_info(python_path)
         Utils::Shebang::RewriteInfo.new(
           %r{^#! ?/usr/bin/(?:env )?python(?:[23](?:\.\d{1,2})?)?( |$)},
           28, # the length of "#! /usr/bin/env pythonx.yyy "
           "#{python_path}\\1",
         )
       end
 
-      def detected_python_shebang(formula = self, use_python_from_path: false)
+      def self.detected_python_shebang(formula = self, use_python_from_path: false)
```

Comment:
This is a mixin module as documented above and this breaks it.

See formulae which invoke `include Language::Python::Shebang` and call `detected_python_shebang`.

Reply:
Reverted in https://github.com/Homebrew/brew/pull/15259, PTAL